backstop_data/engine_scripts/puppet/configure-contributions/setup-personal-campaign.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = async (engine, scenario, viewPort) => {
+  await require('./personal-campaign-pages')(engine, scenario, viewPort);
+
+  const title = await engine.evaluate(el => el.innerHTML, await engine.$('.crm-contribution-contributionpage-pcp-form-block-link_text span.description em'));
+  await engine.goto(title);
```

Comment:
what is this script doing?

Reply:
Getting the URL from the manage contribution's - configure personal Campaign page.

Reply:
Because it is dependent on the pageId param which is dynamic.